src/xrp/xrpEngine.ts

```diff
@@ -208,23 +208,23 @@ export class XrpEngine extends CurrencyEngine<RippleTools> {
       this.error(`Error connecting to server `, e)
       setTimeout(() => {
         if (this.engineOn) {
-          this.startEngine().catch(() => {})
+          this.startEngine().catch(e => console.log(e.message))
         }
       }, 10000)
       return
     }
     this.addToLoop(
       'checkServerInfoInnerLoop',
       BLOCKHEIGHT_POLL_MILLISECONDS
-    ).catch(() => {})
+    ).catch(e => console.log(e.message))
     this.addToLoop('checkAccountInnerLoop', ADDRESS_POLL_MILLISECONDS).catch(
-      () => {}
+      e => console.log(e.message)
     )
     this.addToLoop(
       'checkTransactionsInnerLoop',
       TRANSACTION_POLL_MILLISECONDS
-    ).catch(() => {})
-    await super.startEngine()
+    ).catch(e => console.log(e.message))
+    await super.startEngine().catch(e => console.log(e.message))
```

Comment:
We should delete these `.catch` statements and just `await` all these calls, since the function is `async` anyhow. We really don't want to return success if `addToLoop` has crashed or something.

Reply:
I think this needs a bit of a discussion, I'm not sure if all currencies will throw due to "safe" errors like network errors. We don't want to kill the loop because of a failed network op.

Valid point though but a little out of scope of this PR.